yt/visualization/fixed_resolution.py

```diff
@@ -521,10 +524,6 @@ class OffAxisProjectionFixedResolutionBuffer(FixedResolutionBuffer):
     :class:`yt.visualization.fixed_resolution.FixedResolutionBuffer`
     that supports off axis projections.  This calls the volume renderer.
     """
-    def __init__(self, data_source, bounds, buff_size, antialias = True,
-                 periodic = False):
-        self.data = {}
-        FixedResolutionBuffer.__init__(self, data_source, bounds, buff_size, antialias, periodic)
```

Comment:
Is the `data` attribute not getting used elsewhere?

Reply:
It is initialized in `FixedResolutionBuffer`.